src/components/screens/HomeScreen/AccountItemRow.tsx

```diff
@@ -44,7 +46,7 @@ const AccountItemRow: React.FC<AccountItemRowProps> = ({
   });
 
   const handleViewOnExplorer = async () => {
-    const url = `https://stellar.expert/explorer/public/account/${account.publicKey}`;
+    const url = `${getStellarExpertUrl(NETWORKS.PUBLIC)}/account/${account.publicKey}`;
```

Comment:
@luiz-jung thanks for making the change. I believe the idea here would be passing the `network` variable we have in store so that the url would automatically point to either `public` or `testnet` depending on which network the user is using. Does that make sense? Do you think we could do that here? Similar to what we have on `TransactionDetailsBottomSheet`

cc @danilosilvackl